flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1201,13 +1201,19 @@ bool validInputRange(int16_t min, int16_t max, uint16_t value, uint16_t offset)
  */
 static void applyDeadband(float *value, float deadband)
 {
-	if (fabsf(*value) < deadband)
+	if (deadband < 0.00001f) return;
+
+	if (fabsf(*value) < deadband) {
 		*value = 0.0f;
-	else
-		if (*value > 0.0f)
+	} else {
+		if (*value > 0.0f) {
 			*value -= deadband;
-		else
+		} else {
 			*value += deadband;
+		}
+
+		*value /= 1 - deadband;
```

Comment:
You mean <1?

The runtime checking is an interesting question in general.  THere's plenty of places where we don't do it.  And a lot of times it's not clear what's best to do.  And the strictly "correct" range doesn't really mean it's safe-- if someone sets a deadband of .99, they have almost as big of a problem as if they set it to 1.